app/assets/javascripts/components/uploads/upload_viewer.jsx

```diff
@@ -64,6 +111,7 @@ const UploadViewer = createReactClass({
               <tr>
                 <th>Wiki</th>
                 <th>Article Name</th>
+                <th>{eyeIcon}</th>
```

Comment:
This icon is decent, but I think having a text label is important for the table header.